.eslintrc

```diff
@@ -1,7 +1,7 @@
 {
   "root": true,
   "extends": [
-    "plugin:@wordpress/eslint-plugin/recommended",
+    "plugin:@wordpress/eslint-plugin/recommended-with-formatting",
```

Comment:
Prevents [having to apply](https://github.com/getblocklab/block-lab/pull/673/commits/5186c4242599b98b927e16a6689d382d6aac96c7) all of the Prettier rules, which are mainly formatting